src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,56 @@
+.calendar {
```

Comment:
Move calendar to the center to avoid cell hiding
<img width="770" alt="Screenshot 2023-09-29 at 15 21 16" src="https://github.com/mate-academy/layout_calendar/assets/117587463/0d17e248-f8e6-4904-b170-ce0fb1cddb3c">